### ISSUE TITLE
Crashing When Loading Minecraft

### ISSUE THREAD

**Issue by PowerGamesReal** [Johni0702/BetterPortals#406]
When i Click Play In the Minecraft Launcher it loads up but in the main it doesn't show the full image just the bottom left shows the hole other part of the screen is just black and then when i click the maximize button it just crashes please tell me how to fix this!

**Comment by mcDandy** [comment]
You will need to send crash report. Maybe client.log or other files will be
needed.

Dne po 4. lis 2019 21:03 uživatel PowerGamesReal <notifications@github.com>
napsal:

**Comment by mcDandy** [comment]
So if it crashes without crash report… It is strange. Try updating your graphics drivers. Author might need some details like latest.log(if played again it will be named with date) or debug.log (if it generates). For this I will leave decission on him/her.

**Comment by mcDandy** [comment]
She will at least need to know what version of mods are you using, which version of froge.

**Comment by Johni0702** [comment]
Yeah, `latest.log` and/or `debug.log` (from `.minecraft/logs`) would be great.
Also have a look into `.minecraft/crash-reports`, maybe the crash report is generated but not shown.

Screenshots would also help, specifically of the "unexpected error", of the main menu and, in case you've gotten that far, from in-game.

**Comment by aliginus** [comment]
yes , game shuting down start
https://paste.ee/p/U8P4h

**Comment by Johni0702** [comment]
@aliginus Mekanica is not supported. Use an up-to-date Mekanism instead. Also, don't hijack other people's issues, almost always just open your own.

**Comment by QuantumSD** [comment]
Hi sorry i have another issue with this mod and i posted it on another post, and i have a crash report on that post.
